pkg/dockerregistry/server/auth.go

```diff
@@ -90,7 +91,7 @@ var (
 )
 
 func newAccessController(options map[string]interface{}) (registryauth.AccessController, error) {
-	log.Info("Using Origin Auth handler")
+	log.Info("using Origin Auth handler")
```

Comment:
> This code is part of origin code in which the patterns I've mentioned apply.
 
Yes, but the message appears only in registry's logs. Another reason for having registry in its own repository :smile:.